tests/functionnal/TicketTask.php

```diff
@@ -139,6 +138,18 @@ public function testSchedulingAndRecall() {
       ]);
       $this->integer($task_id)->isGreaterThan(0);
 
+      $this->hasSessionMessage(
+         WARNING, [
+            sprintf(
+               'The user <a href="/glpi/front/user.form.php?id=%s">_test_user</a> is busy at the selected timeframe.<br/>- Ticket task: from %s to %s:<br/><a href=\'/glpi/front/ticket.form.php?id=%s&amp;forcetab=TicketTask$1\'>ticket title</a><br/>',
+               $uid,
+               $date_begin->format('Y-m-d H:i'),
+               $date_end->format('Y-m-d H:i'),
+               $ticketId
+            )
+         ]
+      );
```

Comment:
I would have change planned date to prevent conflict. Conflicts are already checked on `testPlanningConflict()` method.

Reply:
Done